cargo/private/cargo_build_script.bzl

```diff
@@ -87,6 +87,7 @@ def _cargo_build_script_impl(ctx):
     flags_out = ctx.actions.declare_file(ctx.label.name + ".flags")
     link_flags = ctx.actions.declare_file(ctx.label.name + ".linkflags")
     link_search_paths = ctx.actions.declare_file(ctx.label.name + ".linksearchpaths")  # rustc-link-search, propagated from transitive dependencies
+    link_search_prefix = "%s" % (ctx.label.workspace_root) if toolchain.target_arch == "powerpc64le" else ""
```

Comment:
Can you explain this change?

Reply:
@UebelAndre I had to make this change because one of the required C libraries was not getting found at the expected location. A prefix had to be added to the path but only for the power build and not the Intel build. Please allow me some time to recollect the name of the library and other the specifics.

Reply:
@UebelAndre The library in question is **rustix_outline_powerpc64**. 
The path that was going in with -L was **src/backend/linux_raw/arch/outline/release/** 
It should be **external/cui__rustix-0.37.23/src/backend/linux_raw/arch/outline/release** , because the library was at the location **/root/.cache/bazel/_bazel_root/ff51cf56844f3e41f238392435809bdf/external/cui__rustix-0.37.23/src/backend/linux_raw/arch/outline/release/librustix_outline_powerpc64.a**

I hope this makes sense.